### DISCUSSION TITLE
additional backend storage beside git, fs...

### DISCUSSION THREAD

**Discussion by splazit** [gopasspw/gopass#2578] [Ideas]
What are your thoughts on adding another storage backend that is more distributed like Storj?

**Comment by dominikschulz** [comment]
We did have different backends (e.g. supporting HashiCorp Vault), but that diverged too far from the core use case (git+gpg) that we eventually stopped supporting it.

**Comment by splazit** [comment]
I see. Do you see the use case for distributed storage backend + gpg? To me it would be safer than centralized storage.

**Reply by dominikschulz** [reply]
Maybe. If it integrates nicely.
